docs/userGuide/syntax/code.mbdf

```diff
@@ -125,6 +125,25 @@ To add a heading, add the attribute `heading` with the heading text as the value
 </span>
 </include>
 
+Headings support inline MarkDown, except for `Inline Code` and %%Dim%% text styles.
```

Comment:
I think we use 'Markdown' in all other places.
```suggestion
Headings support inline Markdown, except for `Inline Code` and %%Dim%% text styles.
```

Reply:
I see, thank you!